cmds/utk/utk.go

```diff
@@ -56,23 +56,62 @@ package main
 import (
 	"flag"
 	"fmt"
+	"strconv"
+	"strings"
 
 	"github.com/linuxboot/fiano/pkg/log"
+	"github.com/linuxboot/fiano/pkg/uefi"
 	"github.com/linuxboot/fiano/pkg/utk"
 	"github.com/linuxboot/fiano/pkg/visitors"
 )
 
-func init() {
+type config struct {
+	ErasePolarity *byte
+}
+
+func parseArguments() (config, []string, error) {
 	flag.Usage = func() {
 		fmt.Fprintf(flag.CommandLine.Output(), "Usage: utk [flags] <file name> [0 or more operations]\n")
 		flag.PrintDefaults()
 		fmt.Fprintf(flag.CommandLine.Output(), "\nOperations:\n%s", visitors.ListCLI())
 	}
+	erasePolarityFlag := flag.String("erase-polarity", "", "set erase polarity; possible values: '', '0x00', '0xFF'")
+	flag.Parse()
+
+	var cfg config
+
+	switch {
+	case *erasePolarityFlag == "":
+	case strings.HasPrefix(*erasePolarityFlag, "0x"):
+		erasePolarity, err := strconv.ParseUint((*erasePolarityFlag)[2:], 16, 8)
+		if err != nil {
+			return config{}, nil, fmt.Errorf("unable to parse erase polarity '%s': %w", (*erasePolarityFlag)[2:], err)
+		}
+		cfg.ErasePolarity = &[]uint8{uint8(erasePolarity)}[0]
```

Comment:
Is there value in ensuring the erase polarity is either 0 or 0xFF? with this we could supply 0xDE if we wanted to. I'm ok either way

Reply:
If the polarity is invalid, it will panic here:
https://github.com/linuxboot/fiano/pull/358/files#diff-239a5539d7bbec3894ab256b4b20075436aa19b06b5f49995786e36163bdd837R109-R111

Reply:
Ah ok!